telehealth-example/src/main/res/raw/users.json

```diff
@@ -0,0 +1,50 @@
+[
+  {
+    "name": "Luke Young",
+    "custom": {
+      "title": "Product Engineer",
+      "username":"lukeyoung",
```

Comment:
What's the relation between this file and `patient.json` in `assets`? Do we really need both?

Reply:
One is removed, only one is left